postings/views.py

```diff
@@ -0,0 +1,98 @@
+import json
+
+from django.http  import JsonResponse
+from django.views import View
+
+from postings.models import Posting, Comment
+from users.models    import User
+
+class PostingDeatilView(View):
+    # 여기는 데코레이터가...필요가 없나? 비회원도 포스팅 상세를 볼 수는 있으니까.. 필요없음?
+    def get(self, request, posting_id):
+        try:    
+            posting = Posting.objects.get(id = posting_id)
+            posting.hits += 1
+            posting.save()
+            
+            images = [{'image_id' : image.id, 'image_url' : image.image_url}
+                      for image in posting.image_set.all()]
+            user   = User.objects.get(id = posting.user_id)
+        
+            # user관련 내용들 : 유저이름 / 유저사진
+            # likes 좋아요기능 띄워야함!!
+        
+            result = {
+                'posting_id' : posting.id,
+                'title'      : posting.title,
+                'content'    : posting.content,
+                'tags'       : posting.tags.split(','),
+                'size'       : posting.size.name,
+                'residence'  : posting.residence.name,
+                'space'      : posting.space.name,
+                'style'      : posting.style.name,
+                'hits'       : posting.hits,
+                'image_urls' : images,
+                'user_name'  : user.nickname,
+                'user_image' : user.profile_image_url
+            }
+        
+            return JsonResponse({'result' : result}, status=200)
+    
+        except Posting.DoesNotExist:
+            return JsonResponse({'message' : 'POSTING_DOESNOT_EXIST'}, status=400)
```

Comment:
```suggestion
            return JsonResponse({'message' : 'POSTING_DOESNOT_EXIST'}, status=404)
```
does not exist 에러는 보통 404 코드를 사용합니다.